lib/types.ts

```diff
@@ -29,3 +29,12 @@ export interface TokenInfo {
   description?: string
   logoURI?: string
 }
+
+export interface NftsList {
```

Comment:
Shall we call this `NFTList` instead (same way we say `TokenList` instead of `TokensList`)?

Reply:
yes indeed, nice point.

Reply:
Maybe `NFTCollectionList` since it's only about collections?